core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java

```diff
@@ -420,11 +406,15 @@ private String extractRoadAttribute(ReaderWay way, List<String> keys) {
         return Double.parseDouble(value) * factor;
     }
 
-    double stringToMeter(String value) {
+    public static double stringToMeter(String value) {
```

Comment:
Made it public to reuse it elsewhere ... before I thought putting it into OSMTagParser but this does not work, see #939